dev/com.ibm.ws.jmx.connector.client.rest/test/com/ibm/ws/jmx/connector/converter/JSONConverterTest.java

```diff
@@ -286,7 +287,7 @@
 
     private static final String TEST_ATTRIBUTE_NAME = "attributeName";
 
-    private static final String TEST_THROWABLE_STACK_TRACE = "throwable stack trace";
+    private static final String TEST_THROWABLE_MESSAGE = "throwable stack trace";
```

Comment:
The value doesn't really matter, but since it's no longer a stack trace, it'll be good to change the value to "throwable error message"